_layouts/meeting_list.html

```diff
@@ -49,22 +49,22 @@ <h5>
                             - [<a href="https://time.is/{{ item.start | date: "%H%M%p_%-d_%B_%Y_in_Chicago" }}">Your Time</a>]
                             [<a href="https://raw.githubusercontent.com/mpi-forum/mpi-forum.github.io/master/_data/ical_files/{{ item.start | date: "%Y-%m-%d-%H" }}.ics" download>iCAL Event</a>]
                         {% endif %}
-                    </h5>
+                    </h4>
                 {% else %}
-                    <h5>{{ item.start | date: "%-d %B %Y" }} - {{ item.end | date: "%-d %B %Y" }} - {{ item.title }}</h5>
+                    <h4>{{ item.start | date: "%-d %B %Y" }} - {{ item.end | date: "%-d %B %Y" }} - {{ item.title }}</h4>
                     {% if item.links != "no" %}
-                        <h4><table><tr>
+                        <h5><table><tr>
```

Comment:
Personally, I like the higher density and would prefer to take just this change (and the matching tag a few lines down of course).

Reply:
Honestly, the whole website is pretty ugly, so maybe I'm a bad person to make these sorts of decisions.

Reply:
I tried out having both headline and content as `<h5>` and to my mind it reduces readability again. Having the headline in a larger font than the links helps recognizing them as a unit.

I suppose we could tweak the css to enhance visual blocking (adjust margins, etc.) but I have no real bootstrap experience, so that was not straight-forward for me.

Reply:
Ok. I don't have strong enough opinions to do a bunch of CSS stuff. I don't know anything about that either. I'll just merge this then.